packages/next/src/server/web/spec-extension/revalidate.ts

```diff
@@ -22,30 +21,32 @@ export function revalidateTag(tag: string) {
 }
 
 /**
- * @deprecated this function has been deprecated in favor of expirePath()
  * This function allows you to purge [cached data](https://nextjs.org/docs/app/building-your-application/caching) on-demand for a specific path.
  *
  * Read more: [Next.js Docs: `revalidatePath`](https://nextjs.org/docs/app/api-reference/functions/revalidatePath)
  */
 export function revalidatePath(originalPath: string, type?: 'layout' | 'page') {
-  return expirePath(originalPath, type)
+  return unstable_expirePath(originalPath, type)
 }
 
 /**
  * This function allows you to purge [cached data](https://nextjs.org/docs/app/building-your-application/caching) on-demand for a specific cache tag.
  *
- * Read more: [Next.js Docs: `expireTag`](https://nextjs.org/docs/app/api-reference/functions/expireTag)
+ * Read more: [Next.js Docs: `unstable_expireTag`](https://nextjs.org/docs/app/api-reference/functions/unstable_expireTag)
  */
-export function expireTag(...tags: string[]) {
-  return revalidate(tags, `expireTag ${tags.join(', ')}`)
+export function unstable_expireTag(...tags: string[]) {
+  return revalidate(tags, `unstable_expireTag ${tags.join(', ')}`)
 }
 
 /**
  * This function allows you to purge [cached data](https://nextjs.org/docs/app/building-your-application/caching) on-demand for a specific path.
  *
- * Read more: [Next.js Docs: `expirePath`](https://nextjs.org/docs/app/api-reference/functions/expirePath)
+ * Read more: [Next.js Docs: `unstable_expirePath`](https://nextjs.org/docs/app/api-reference/functions/unstable_expirePath)
  */
-export function expirePath(originalPath: string, type?: 'layout' | 'page') {
+export function unstable_expirePath(
```

Comment:
should we revert this back to `revalidatePath` and add a new duplicate handler for `unstable_expirePath`, since the functions will most likely need different handling, and that way we can fix the warning below (it currently mentions `revalidatePath`